io/disk/ssd/blkdiscard.py

```diff
@@ -48,7 +49,8 @@ def setUp(self):
         smm = SoftwareManager()
         if not smm.check_installed("util-linux"):
             self.cancel("blkdiscard is needed for the test to be run")
-        self.disk = self.params.get('disk', default='/dev/nvme0n1')
+        device = self.params.get('disk', default=None)
```

Comment:
We need to handle the case where device=None, and get_absolute_disk_path(None) could fail/error out.

Reply:
Similarly for all files

Reply:
@narasimhan-v Done, have taken care in latest commits